pkg/plugins/resources/xml/condition.go

```diff
@@ -17,6 +17,7 @@ func (x *XML) ConditionFromSCM(source string, scm scm.ScmHandler) (bool, error)
 
 	if scm != nil {
 		x.spec.File = joinPathWithWorkingDirectoryPath(x.spec.File, scm.GetDirectory())
+		logrus.Debugf("Relative path detected: changing to absolute path from SCM: %q", x.spec.File)
```

Comment:
This change is not related to this PR: wrong rebase?

Reply:
I don't think it's from a wrong rebase, the related commit is "chore: add debug logs"

I agree it's not directly related to this PR, I must have added it last september wanting to do good.

I can remove it no problem, but as it's only additional logs, WDYT keeping it?

Reply:
It changes a file unrelated to the PR: I don't mind a PR explaining the intent (since it's done twice) but it can't stay in this one. (same for the other) otherwise the whole PR process (with labels and changelog) would not make sense

Reply:
Moved here: https://github.com/updatecli/updatecli/pull/1119